kotlin-native/runtime/src/main/kotlin/kotlin/native/internal/FloatingPointParser.kt

```diff
@@ -284,98 +353,4 @@ object FloatingPointParser {
         }
         return first == '0' && (second == 'x' || second == 'X')
     }
-
-    /**
-     * Returns the closest double value to the real number in the string.
-     *
-     * @param string the String that will be parsed to a floating point
-     * @return the double closest to the real number
-     * @exception NumberFormatException if the String doesn't represent a double
-     */
-    fun parseDouble(string: String): Double {
```

Comment:
This is a code duplication, that is unified already in one single _inline generic_ method for all types. 
This logic should be in one place, so we will reduce the number of potential issues.